src/ruby.ts

```diff
@@ -122,6 +122,11 @@ export class Ruby {
       cwd: this.workingFolder,
     });
 
+    if (result.stdout === "") {
```

Comment:
Could there be any whitespace in `stdout`? If so, we would be safer with a [`trim()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim):
```suggestion
    if (result.stdout.trim() === "") {
```

Reply:
I don't think so, but it doesn't hurt either.